src/features/resources/ResourceItem.tsx

```diff
@@ -53,32 +75,117 @@ const ResourceItem = ({ resource, event }: { resource: Types.Resource; event: an
-          }
-        }}
-      >
-        {IconLookup(resource.iconName, themeContext.features.resources.icon.color)}
-        <ListItemContentLinkName>{resource.title}</ListItemContentLinkName>
-      </ListItemResourceLink>
-      <Checkbox
-        icon={<Icon icon={faHeart} />}
-        checkedIcon={<Icon icon={faHeart} color="#d73f09" />}
-        value={resource.id}
-        checked={favs}
-        onChange={handleChange}
-        inputProps={{
-          'aria-label': favoriteLabelText(favs),
-        }}
-      />
-    </ListItemFlex>
+    <>
+      <ListItemFlex>
+        <ListItemResourceLink
+          onClick={() => {
+            // if resource's IT system has an error, open the dialog box
+            if (itSystemError) {
+              open();
+            }
+            // else, open link
+            else {
+              window.open(resource.link);
+              close();
+            }
+            event();
+            if (!resource.excludeTrending) {
+              TrendingEvent(resource, user.data);
+            }
+          }}
+        >
+          {IconLookup(resource.iconName, themeContext.features.resources.icon.color)}
+          <ListItemContentLinkName>{resource.title}</ListItemContentLinkName>
+          {itSystemError && (
+            <Icon
+              fontSize={fontSize[18]}
+              icon={faExclamationCircleSolid}
+              color={themeContext.features.itStatus.item.icon.partialOutage}
+              style={{ marginLeft: '5px', fontSize: '18px!important' }}
+              data-testid="warning-icon"
+            />
+          )}
+        </ListItemResourceLink>
+
+        <Checkbox
+          icon={<Icon icon={faHeart} />}
+          checkedIcon={<Icon icon={faHeart} color="#d73f09" />}
+          value={resource.id}
+          checked={favs}
+          onChange={handleChange}
+          inputProps={{
+            'aria-label': favoriteLabelText(favs),
+          }}
+        />
+      </ListItemFlex>
+
+      {showDialog && (
+        <MyDialog
+          isOpen={showDialog}
+          onDismiss={close}
+          aria-labelledby="message-title"
+          style={{ marginTop: '30vh' }}
+        >
+          <CloseButton onClick={close} />
+          <div>
+            <Icon
+              fontSize={fontSize[26]}
+              icon={faExclamationCircleHollow}
+              color={themeContext.features.itStatus.item.icon.partialOutage}
+              style={{ display: 'inline-block', paddingRight: '5px' }}
+            />{' '}
+            <h2 id="message-title" style={{ fontSize: fontSize[18], display: 'inline-block' }}>
+              This resource may be unavailable.
+            </h2>
+          </div>
+          <DateContainer>
+            {resource.title} •{' '}
+            {systemCheckedAt.toLocaleString('en-US', {
+              hour: 'numeric',
+              minute: 'numeric',
+              hour12: true,
+            }) +
+              ' at ' +
```

Comment:
I think this should be 'on' is that right.. the resulting text would be something like "4:20pm on April 21, 2021" right?